coderd/audit.go

```diff
@@ -64,7 +64,21 @@ func (api *API) auditLogs(rw http.ResponseWriter, r *http.Request) {
 		filter.Username = ""
 	}
 
-	dblogs, err := api.Database.GetAuditLogsOffset(ctx, filter)
+	// Use the same filters to count the number of audit logs
+	count, err := api.Database.CountAuditLogs(ctx, database.CountAuditLogsParams{
```

Comment:
It's going to be very easy for someone to forget to update this in both places. Maybe you should make the `searchquery.AuditLogs` package return both of these params types so they're constructed in the same place

Or add a comment in both places

Reply:
Moved to searchquery